InvasivesBC/Utilities/Notification/Notifications.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Notifications.swift
+//  InvasivesBC
+//
+//  Created by Amir Shayegh on 2020-04-20.
+//  Copyright © 2020 Amir Shayegh. All rights reserved.
+//
+
+import Foundation
+
+extension Notification.Name {
+    static let screenOrientationChanged = Notification.Name("screenOrientationChanged")
```

Comment:
Maintain a similar naming convention. The recommended, static const variable should start with a Captial letter.